src/Compilers/Core/Portable/Emit/EmitOptions.cs

```diff
@@ -122,6 +130,37 @@ public sealed class EmitOptions : IEquatable<EmitOptions>
         {
         }
 
+        // 2.7 BACKCOMPAT OVERLOAD -- DO NOT TOUCH
+        public EmitOptions(
+            bool metadataOnly,
+            DebugInformationFormat debugInformationFormat,
+            string pdbFilePath,
+            string outputNameOverride,
+            int fileAlignment,
+            ulong baseAddress,
+            bool highEntropyVirtualAddressSpace,
+            SubsystemVersion subsystemVersion,
+            string runtimeMetadataVersion,
+            bool tolerateErrors,
+            bool includePrivateMembers,
+            ImmutableArray<InstrumentationKind> instrumentationKinds)
+            : this(
+                  metadataOnly,
+                  debugInformationFormat,
+                  pdbFilePath,
+                  outputNameOverride,
+                  fileAlignment,
+                  baseAddress,
+                  highEntropyVirtualAddressSpace,
+                  subsystemVersion,
+                  runtimeMetadataVersion,
+                  tolerateErrors,
+                  includePrivateMembers,
+                  instrumentationKinds: ImmutableArray<InstrumentationKind>.Empty,
```

Comment:
>instrumentationKinds [](start = 18, length = 20)

I think the value of this parameter should not be `Empty`, but rather the value of the incoming parameter.

Reply:
You are right. Apparently we are missing code coverage here.